src/backend/utilities/configuration.ts

```diff
@@ -52,6 +52,15 @@ if (!discord.clientId || !discord.clientSecret) {
   throw new Error('No discord client credentials provided');
 }
 
+const github = {
+  clientId: env.GITHUB_CLIENT_ID as string,
+  secret: env.GITHUB_SECRET,
```

Comment:
Let’s remind ourselves it’s secret
```suggestion
  secret: env.SECRET_GITHUB,
```